db/flush_job.cc

```diff
@@ -228,6 +228,9 @@ Status FlushJob::Run(LogsWithPrepTracker* prep_tracker,
     prev_cpu_read_nanos = IOSTATS(cpu_read_nanos);
   }
   Status mempurge_s = Status::NotFound("No MemPurge.");
+  if (switched_to_mempurge) {
+    *switched_to_mempurge = 0;
```

Comment:
Do we need to set it or it will be 0 by default?

Reply:
It is `0` by default, I added this out of paranoia. It's probably cleaner to remove this you are right, thanks for your input.